proxy/README.md

```diff
@@ -110,7 +110,7 @@ The proxy can output log messages on stderr, which are automatically
 handled by systemd and can be viewed with:
 
 ```
-journalctl -u cc-proxy -f
+sudo journalctl -u cc-proxy -f
```

Comment:
OK, can do. I looked at the one above at the time so kept 'consistency' with that one, but not I guess the whole doc or project ;-) - will fix and re-submit